website/docs/installation.mdx

```diff
@@ -16,28 +19,39 @@ pip install h2o-wave
 
 To install in a virtual environment (recommended):
 
+<Tabs groupId="operating-systems">
+
+<TabItem value="lin" label="Linux">
+
 ```shell
 python3 -m venv venv
 source venv/bin/activate
 pip install h2o-wave
 ```
 
-To install in a virtual environment (Windows):
+</TabItem>
+
+<TabItem value="win" label="Windows">
 
 ```shell
-python3 -m venv venv
-venv\Scripts\activate
+python -m venv venv
+cd venv/Scripts/
+./activate.bat(CMD) or ./Activate.ps1(poweshell)
 pip install h2o-wave
 ```
+</TabItem>
 
-To install using Conda:
+<TabItem value="conda" label="Conda">
```

Comment:
Let's not mix Conda with virtual env section. A better option would be to make a new PyPi/Conda tabs group above. Also do not forget about including the info bar saying wave server is not packaged into Conda  - as conda in virtual env has.

Reply:
Hi
![Screenshot 2023-07-07 100521](https://github.com/h2oai/wave/assets/60529828/6e41d5c0-132c-435f-9a04-ea224b1f218d)

Does this work for this issue

Reply:
Yes, looking good. One small thing, let's go with `MacOS` instead of `MacOs` everywhere.

Reply:
Noted